extensions/spark/kyuubi-spark-authz/src/test/resources/sparkSql_hive_jenkins.json

```diff
@@ -1151,6 +1151,67 @@
       "guid": "b3f1f1e0-2bd6-4b20-8a32-a531006ae151",
       "isEnabled": true,
       "version": 1
+    },
+    {
+      "service": "hive_jenkins",
```

Comment:
Use or modify existing policy, please.

Reply:
Reconfirmed that no existing policy for the perm view.